tests/roles/common_defaults/defaults/main.yaml

```diff
@@ -11,3 +11,7 @@ use_no_log: false
 
 # Whether the adopted node will host compute services
 compute_adoption: true
+
+# Whether to adopt Octavia
+enable_octavia: true
```

Comment:
sanity check. do we really mean to default to on here ? 
on the other side in rdo-jobs you default false
```
            -e "enable_octavia={{ enable_octavia | default('false') }}"
```

Reply:
It has been disabled because of a CIX: https://issues.redhat.com/browse/OSPCIX-420 . It crashed tempest. I think we should keep it enabled per default if possible in order to test it upsteam at least.
@cescgina

Reply:
It crashed tempest because Octavia was disabled, enabling it during the deployment and the adoption will fix it